### DISCUSSION TITLE
Sidebar multiple

### DISCUSSION THREAD

**Discussion by dsultanr** [bootstrap-vue/bootstrap-vue#6863] [Q&A] [answered]
Is it possible to have a multiple sidebars like this?

![image](https://user-images.githubusercontent.com/1683476/145097588-99ead3eb-ddf7-48a5-b90f-e531279ac423.png)

**Comment by Hiws** [comment, marked as answer]
Doesn't you screenshot literally show that it's possible? 🤔

**Reply by dsultanr** [reply]
I made it work by "hacking" the DOM directly. Is it possible to create a multiple sidebar slots and fine-tune their z-index and width attributes? 

Googled a while and unfortunatelly couldn't find any examples of multiple sidebars...

**Reply by Hiws** [reply]
`<b-sidebar>` has both a `width` and `z-index` prop, so you can use those.

```html
<b-sidebar z-index="2000" width="400px">Sidebar 1</b-sidebar>
<b-sidebar z-index="2001">Sidebar 2</b-sidebar>
```

also, by default AFAIK, if you don't define a `z-index`, then the sidebar opened last will be on top.
